src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -6,18 +6,117 @@
 import codeu.model.store.basic.ConversationStore;
 import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+import java.util.List;
+import java.util.HashMap;
+import java.util.UUID;
+import java.time.Instant;
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /**
  * Servlet class responsible for user registration.
  */
 public class AdminServlet extends BaseServlet {
+  /** Set up state for compiling statistics */
+  @Override
+  public void init() throws ServletException {
+    super.init();
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+    request.setAttribute("numUsers", numUsers());
+    request.setAttribute("numConvos", numConvos());
+    request.setAttribute("numMessages", numMessages());
+    request.setAttribute("mostActiveUser", mostActiveUser());
+    request.setAttribute("newestUser", newestUser());
+    request.setAttribute("wordiestUser", wordiestUser());
     request.getRequestDispatcher("/WEB-INF/view/admin.jsp").forward(request, response);
   }
+
+  /** Returns number of users in database */
+  private int numUsers() {
+    return this.userStore.getUsers().size();
+  }
+
+  /** Returns number of conversations in database */
+  private int numConvos() {
+    return this.conversationStore.getConversations().size();
+  }
+
+  /** Returns number of messages in database */
+  private int numMessages() {
+    return this.messageStore.getMessages().size();
+  }
+
+  /** Calculated by user with most messsages sent and most conversations started */
+  private String mostActiveUser() {
+    // Mapping User's UUID to the frequency of their contributions
+    HashMap<UUID, Integer> frequencyMap = new HashMap<UUID, Integer>();
+    String mostActiveUserName = "No messages/conversations posted";
+    List<Conversation> conversations = this.conversationStore.getConversations();
```

Comment:
This one has a similar problem: excessive copying. It's harder to address this problem here, so I'll let it slide for now, but we would probably need to find ways to optimize this later.